packages/tslint-config/scripts/postpublish.js

```diff
@@ -0,0 +1,14 @@
+const postpublish_utils = require('../../../scripts/postpublish_utils');
```

Comment:
Can we have this script in one place and symlink to it?

Reply:
We could... my hunch thought is that these will eventually diverge for different sub-packages.